app/assets/javascripts/angular/controllers/my_books.js.erb

```diff
@@ -0,0 +1,61 @@
+app.controller('MyBooksCtrl', ['$scope', '$http', '$location', '$auth', function($scope, $http, $location, $auth){
+
+  var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
+
+  $http.get(url + '/mybooks').success(function(response){
+    console.log(response);
+    $scope.items = response;
+  });
+
+  $scope.book = {};
+  $scope.form = {};
+
+  $scope.form.create = function(){
+    var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
+
+    var data = {
+      book: $scope.book
+    };
+
+    $http.post(url + '/books', data).success(function(response){
+      console.log(response);
+    });
+  };
+
+  $scope.apiSearch = function(keyEvent, value) {
```

Comment:
This is the only place where I use this function. I will need factory somewhere else though lol. Thanks for the advice!!